awx/main/models/projects.py

```diff
@@ -597,10 +620,7 @@ def preferred_instance_groups(self):
     def save(self, *args, **kwargs):
         added_update_fields = []
         if not self.job_tags:
-            job_tags = ['update_{}'.format(self.scm_type)]
-            if self.job_type == 'run':
-                job_tags.append('install_roles')
-                job_tags.append('install_collections')
+            job_tags = ['update_{}'.format(self.scm_type), 'install_roles', 'install_collections']
```

Comment:
Let me get this out of the way:

With role/collection caching in place, it no longer makes _any_ sense to have 2 separate tags for these things. It should just be 1 tag, say "install_content". If you follow all the code here, you will see that one of them is never used without the other.

I am not consolidating those tags right now, because `job_tags` are a primary part of the model and something that the user sees. In the effort to maintain behavior consistency I'm not changing it now, but after this PR, we could/should make that change for `devel` branch.